src/main/java/hu/eszter/bokkon/model/participants/Farmer.java

```diff
@@ -1,29 +1,39 @@
 package hu.eszter.bokkon.model.participants;
 
-import hu.eszter.bokkon.model.animal.Animal;
+import hu.eszter.bokkon.model.animal.*;
 
-import java.util.*;
+import java.security.SecureRandom;
+import java.util.LinkedHashMap;
+import java.util.Map;
 
 /**
  * Farmer is the player, who can collect animals. 2-4 players can play in 1 game.
  */
 public class Farmer implements MoveAnimal {
 
     private final String name;
-    private final Random random = new Random();
-    private Map<Animal, Integer> farmerLiveStock = new LinkedHashMap<>();
+    private final SecureRandom random = new SecureRandom();
+    private Map<Animal, Integer> animalStock;
     private Map<Animal, Map<Animal, Double>> actualPossibleChanges = new LinkedHashMap<>();
 
     public Farmer(String name) {
         this.name = name;
+        this.animalStock = new LinkedHashMap<>();
+        animalStock.put(new Rabbit(), 0);
+        animalStock.put(new Sheep(), 0);
+        animalStock.put(new Pig(), 0);
+        animalStock.put(new Cow(), 0);
+        animalStock.put(new Horse(), 0);
+        animalStock.put(new SmallDog(), 0);
+        animalStock.put(new BigDog(), 0);
     }
 
     public String getName() {
         return name;
     }
 
-    public Map<Animal, Integer> getFarmerLiveStock() {
-        return farmerLiveStock;
+    public Map<Animal, Integer> getAnimalStock() {
```

Comment:
This method could be unified with AnimalBaseStock.getLiveStock through MoveAnimal interface, if I am not mistaken both serves the same purpose.